assets/js/hooks/index.ts

```diff
@@ -682,21 +682,52 @@ export const BlurDataclipEditor = {
   },
 } as PhoenixHook;
 
-export const ScrollToBottom = {
+export const ScrollToMessage = {
   mounted() {
-    this.scrollToLastElement();
+    this.handleScroll();
   },
+  
   updated() {
-    this.scrollToLastElement();
+    this.handleScroll();
+  },
+  
+  handleScroll() {
+    const targetMessageId = this.el.dataset['scrollToMessage'];
+    
+    if (targetMessageId) {
+      this.scrollToSpecificMessage(targetMessageId);
+    } else {
+      this.scrollToBottom();
+    }
   },
-  scrollToLastElement() {
-    this.el.lastElementChild &&
-      this.el.lastElementChild.scrollIntoView({
-        behavior: 'smooth',
-        block: 'start',
+  
+  scrollToSpecificMessage(messageId: string) {
```

Comment:
Just thinking out loud. letting this function take the selector instead will make it more generic. that's if we'll want to reuse this same scrolling.